webcompat/static/css/development/components/report-form.css

```diff
@@ -15,6 +15,19 @@
     text-align: center;
     justify-content: center;
     flex-wrap: wrap;
+    margin-top: 1em;
+  }
+
+  .wc-ReportForm .wc-UploadForm {
+    border: dashed 1px rgb(204, 204, 204);
+    border-radius: 20px;
+  }
+
+  .wc-ReportForm .r-Grid-cell {
```

Comment:
re-reading my comment 🤡 I feel we are entering a style/css philosophical discussion. So to avoid this, I would encourage @magsout to suggest a way to organize it as he's the owner and master of the templates. 👍